### DISCUSSION TITLE
Table word wrap.

### DISCUSSION THREAD

**Discussion by ConnorY97** [cotes2020/jekyll-theme-chirpy#1412] [Q&A] [answered]
Is it possible to get table contents to wrap rather than having a scroll bar?

Thanks!

**Comment by ConnorY97** [comment]
I have done some more investigation and still having some issues. 

I have overridden the Style sheet explained [here](https://chirpy.cotes.page/posts/getting-started/#customizing-stylesheet). but I am still finding my changes overwritten: 
![image](https://github.com/cotes2020/jekyll-theme-chirpy/assets/47731551/89417d4c-da27-4770-b603-833776ddeceb)
Any help would be much appreciated!

**Comment by kungfux** [comment, marked as answer]
You can override table cell style in `assets\css\jekyll-theme-chirpy.scss` as following:

```scss
%table-cell {
  white-space: normal;
}
```

Here is the result:
![image](https://github.com/cotes2020/jekyll-theme-chirpy/assets/10548881/92793b0f-d329-47d8-a93d-bb22f3d4d292)

**Comment by ConnorY97** [comment]
Hi, 

Thank you, that worked!

**Reply by kungfux** [reply]
Hi @ConnorY97 

That's great 👍

**Comment by SakharamS** [comment]
---

@import "main";

%table-cell {
    white-space: normal;
  }

--- This is the complete content of the file `assets\css\jekyll-theme-chirpy.scss` which is working for me. Thanks @kungfux
